src/Neo.CLI/CLI/MainService.Plugins.cs

```diff
@@ -90,6 +90,26 @@ private static async Task<Stream> DownloadPluginAsync(string pluginName, Version
                     s["tag_name"]!.GetValue<string>() == $"v{pluginVersion.ToString(3)}" &&
                     s["prerelease"]!.GetValue<bool>() == prerelease) ?? throw new Exception($"Could not find Release {pluginVersion}");
 
+            if (jsonRelease == null)
```

Comment:
`jsonRelease ` is not nul here, so it's still installing default plugins version in v3.7.1